os/20211014-s390x-secure-execution.md

```diff
@@ -0,0 +1,190 @@
+# IBM Z Secure Execution
+---
+
+# Theory of operation
+
+IBM Secure Execution for Linux® is a hardware-based security technology that is
+built into the IBM z15™ and LinuxONE III generation systems. It is designed to
+provide scalable isolation for individual workloads to help protect them from not
```

Comment:
it is isolation among Guest so that nobody can break memory boundaries among two qemu processes. it is workload somewhat but specifically packed in KVM guests.